{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/__init__.py

```diff
@@ -1,5 +1,8 @@
 """The main module for {{ cookiecutter.project_name }}."""
 
-from importlib.metadata import version
+from importlib.metadata import PackageNotFoundError, version
 
-__version__ = version(__name__)
+try:
+    __version__ = version("MUSE_OS")
+except PackageNotFoundError:
+    pass
```

Comment:
Rather than silently ignoring the exception, wouln't it be better to give it a default value - eg:
```suggestion
    __version__ = "unknown"
```
Or something along those lines?